README.md

```diff
@@ -326,6 +339,12 @@ JSON.stringify(#{ a: #[1, 2, 3] }); // '{"a":[1,2,3]}'
 JSON.stringify(#[true, #{ a: #[1, 2, 3] }]); // '[true,{"a":[1,2,3]}]'
 ```
 
+## JSON.parseImmutable
```

Comment:
should we also clarify what `JSON.stringify` does? Does it output JSON object syntax, or fail?

Reply:
presumably, output JSON object syntax is the correct answer, since it can then be consumed again by `JSON.parseImmutable`, but want to verify your thoughts.

Reply:
Yes it should output JSON object syntax